docs/upgrade.md

```diff
@@ -1,6 +1,14 @@
-# Upgrading to PyQIR 0.7
+# Upgrading PyQIR
 
-## Packages
+## PyQIR 0.8
+
+### Packages
+
+PyQIR 0.7 was the last version of PyQIR to support the `pyqir-evaluator`. For analyzing produced QIR, the `pyqir-parser` project can now handle most of what the `pyqir-evaluator` was used for. Simulation of QIR is now available via the [`qir-runner`](https://github.com/qir-alliance/qir-runner) sparse simulator.
```

Comment:
I would just call them the "evaluator" and "parser" now, since `pyqir-evaluator` and `pyqir-parser` are pre-PyQIR 0.7 terms.